README.md

```diff
@@ -199,7 +199,15 @@ The following configurations are set using either environment variables, setting
 | httpAgent/httpsAgent | Not supported in the shim or Azure Monitor OpenTelemetry. |
 | enableWebInstrumentation | Not currently supported in the shim, but is in Azure Monitor OpenTelemetry as `enableBrowserSdkLoader`. |
 | webInstrumentationConnectionString | Not supported in the shim, but is in Azure Monitor OpenTelemetry as `browserSdkLoaderConnectionString`. |
-| webInstrumentationConfig | Not currently supported by the shim, but is in Azure Monitor OpenTelemetry as `browserSdkLoaderConfig`. |
+| webInstrumentationConfig | Not currently supported by the shim, or Azure Monitor OpenTelemetry. |
+| enableAutoCollectPerformance | Not supported in the shim. |
+| enableAutoCollectConsole | Not supported in the shim. |
+| enableAutoCollectExternalLoggers | Not supported in the shim. |
```

Comment:
Should be supported in shim

Reply:
I see performance counters will be, and Console + Winston are not supported in OTel yet, so I'll keep those instrumentations here. But I believe we need to kill `webInstrumentationConfig` since configuring the browser SDK loader is no longer supported in the distro.

Reply:
Yeah I didn't say anything about webInstrumentationConfig, we should deprecate that one

Reply:
It's in the highlighted bit in the UI on Github on my side, sorry for the confusion!